tests/css_check_test.js

```diff
@@ -8,8 +8,15 @@ const getLocator = async (string) => {
            return res.body;
         }else throw new Error('Something is wrong: '+res);
             
-    })
-    const locator = response.split(' ')[1].split('{')[0]; 
+    });
+    console.log("css from file: " + response);
+    let locator;
+    try {
+        locator = response.split('{')[0].trim().split(' ')[1]; 
+    } catch (error) {
```

Comment:
Isn't this identical in impact to the previous code? We catch the error, but then throw the same error again